include/pros/rotation.h

```diff
@@ -0,0 +1,187 @@
+ * failed, setting errno.
+ */
+int32_t rotation_reset(uint8_t port);
+
+/**
+ * Set the rotation sensor position reading to a desired rotation value.
+ *
+ * This function uses the following values of errno when an error state is
+ * reached:
+ * ENXIO - The given value is not within the range of V5 ports (1-21).
+ * ENODEV - The port cannot be configured as an Rotation Sensor
+ *
+ * \param port
+ *		  The V5 Rotation Sensor port number from 1-21
+ * \param position 
+ * 		  The position in terms of ticks
+ * \return 1 if the operation was successful or PROS_ERR if the operation
+ * failed, setting errno.
+ */
+int32_t rotation_set_position(uint8_t port, uint32_t position);
+
+/**
+ * Resets the rotation sensor position to 0.
+ *
+ * This function uses the following values of errno when an error state is
+ * reached:
+ * ENXIO - The given value is not within the range of V5 ports (1-21).
+ * ENODEV - The port cannot be configured as an Rotation Sensor
+ *
+ * \param port
+ *		  The V5 Rotation Sensor port number from 1-21
+
+ * \return 1 if the operation was successful or PROS_ERR if the operation
+ * failed, setting errno.
+ */
+int32_t rotation_reset_position(uint8_t port);
+
+/**
+ * Get the Rotation sensor's current position.  Specifically, in
+ * terms of hundreths of degrees.
+ *
+ * This function uses the following values of errno when an error state is
+ * reached:
+ * ENXIO - The given value is not within the range of V5 ports (1-21).
+ * ENODEV - The port cannot be configured as an Rotation Sensor
+ *
+ * \param  port
+ * 				 The V5 Rotation Sensor port number from 1-21
+ * \return The position value or PROS_ERR_F if the operation failed, setting
+ * errno.
+ */
+int32_t rotation_get_position(uint8_t port);
+
+/**
+ * Get the Rotation sensor's current velocity in hundreths of
+ * degrees per second.
+ *
+ * This function uses the following values of errno when an error state is
+ * reached:
+ * ENXIO - The given value is not within the range of V5 ports (1-21).
+ * ENODEV - The port cannot be configured as an Rotation Sensor
+ *
+ * \param  port
+ * 				 The V5 Rotation Sensor port number from 1-21
+ * \return The velocity value or PROS_ERR_F if the operation failed, setting
+ * errno.
+ */
+int32_t rotation_get_velocity(uint8_t port);
+
+/**
+ * Get the Rotation sensor's current position in terms of an angle
+ * measured in hundreths of degrees.
+ * 
+ * This function uses the following values of errno when an error state is
+ * reached:
+ * ENXIO - The given value is not within the range of V5 ports (1-21).
+ * ENODEV - The port cannot be configured as an Rotation Sensor
+ *
+ * \param  port
+ * 				 The V5 Rotation Sensor port number from 1-21
+ * \return The angle value or PROS_ERR_F if the operation failed, setting
+ * errno.
+ */
+int32_t rotation_get_angle(uint8_t port);
+
+/**
+ * Sets if the rotation sensor's positive/negative direction is reversed or not.
```

Comment:
```suggestion
 * Set the rotation sensor's direction reversed flag
```

Reply:
if you think this needs extra clarification, again: feel free to put it in another line